lib/ret_web/controllers/page_controller.ex

```diff
@@ -229,25 +229,32 @@ defmodule RetWeb.PageController do
   end
 
   def render_index(conn) do
-    {app_config, app_config_script, app_config_csp} = generate_app_config()
+    if conn |> get_req_header("x-original-method") |> Enum.at(0) === "HEAD" do
```

Comment:
haha, this hack lives. i'd probably refactor this into two functions matched on the header value.

Reply:
heh, some day I'll learn to resist `if` statement muscle memory